src/components/HeroSection/HeroSection.jsx

```diff
@@ -42,23 +43,15 @@ export default function HeroSection() {
 						</Grid>
 						<Grid container direction="row" justify="space-between" item>
 							<Grid item xs={6}>
-								<Button
-									className={`${classes.btn} ${classes.foundBtn}`}
-									variant="contained" 
-									size="large"
-									>
-									SUBMIT FOUND ITEMS
-								</Button>
-							</Grid>
-							<Grid item xs={6}>
-								<Button
-									className={`${classes.btn} ${classes.lostBtn}`}
-									variant="contained"
-									size="large"
-
-									>
-									SUBMIT LOST ITEMS
-								</Button>
+								<Link to="/form" className={classes.link}>
+									<Button
+										className={`${classes.btn} ${classes.foundBtn}`}
+										variant="contained" 
+										size="large"
+										>
+										SUBMIT FOUND/LOST ITEMS
+									</Button>
+								</Link>
```

Comment:
Indentation looks off here can we fix that?
Also, the text should be using i18next for translations!